src/commands/new.ts

```diff
@@ -486,14 +486,17 @@ export default {
     const renameSpinnerMsg = `Getting those last few details perfect`
     startSpinner(renameSpinnerMsg)
 
+    const boilerplateBundleIdentifier = "com.helloworld"
     await renameReactNativeApp(
       toolbox,
       "HelloWorld",
       projectName,
-      "com.helloworld",
+      boilerplateBundleIdentifier,
       bundleIdentifier,
     )
 
+    await replaceMaestroBundleIds(toolbox, boilerplateBundleIdentifier, bundleIdentifier)
```

Comment:
Felt that this being separate utility fn was just cleaner

Reply:
Yeah I like this being semerate, makes it easier to read the main command